src/plugins/mergeCells/calculations/autofill.js

```diff
@@ -379,20 +379,22 @@ class AutofillCalculations {
     const columns = { min: null, max: null };
 
     arrayEach(changes, (change) => {
-      if (rows.min === null || change[0] < rows.min) {
-        rows.min = change[0];
+      const [rowIndex, columnIndex] = change.map((el, i) => (i === 1 ? this.plugin.hot.propToCol(el) : el));
```

Comment:
Can I ask you to use `arrayMap` helper? Our function is a faster variant for built-in API and more performant stable.